ddtrace/internal/telemetry/writer.py

```diff
@@ -66,6 +67,9 @@ def __init__(self, agent_url=None):
         # _sequence is a counter representing the number of requests sent by the writer
         self._sequence = 1  # type: int
 
+    def __del__(self):
```

Comment:
Is this needed?

Reply:
Yes and no: this method is never going to be called because the forksafe hook is going to hold a reference to this object forever.

Reply:
Since a reference to this object will live forever we shouldn't need to deregister the this hook